eventmesh-common/src/main/java/org/apache/eventmesh/common/protocol/tcp/codec/Codec.java

```diff
@@ -108,14 +116,14 @@ public void decode(ChannelHandlerContext ctx, ByteBuf in, List<Object> out) thro
 
                 final int length = in.readInt();
                 final int headerLength = in.readInt();
-                final int bodyLength = length - 8 - headerLength;
+                final int bodyLength = length - headerLength;
```

Comment:
this line need to change:
`final int bodyLength = length - CONSTANT_MAGIC_FLAG.length - VERSION.length - headerLength;`